.github/workflows/tenzir.yaml

```diff
@@ -702,6 +702,9 @@ jobs:
       - name: Install
         run: |
           cmake --install "$BUILD_DIR"
+      - name: Run Better Integration Tests
```

Comment:
Srsly?

Reply:
This is temporary, I already added a CTest integration that covers this. I'll push the se changes as part of my review.